classes/classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students = student
+    student.classroom = self
+  end
```

Comment:
- [ ] I see a little issue with the `add_student` method, on line 11, you are assigning the student to the `@students` instance variable instead of pushing the student into the `@students` array. Like so 👇🏾 

```ruby
  def add_student(student)
    @students << student #  student is pushed into the array
    student.classroom = self
  end
```

Please fix that.